examples/extended/create.sh

```diff
@@ -5,7 +5,7 @@ kubectl.sh create -f existing_job.yaml
 
 kubectl.sh create -f ../../manifests/appcontroller.yaml
 
-sleep 10
+sleep 17
 
```

Comment:
Can't we pool k8s api instead?

Or at least, maybe could we add comment there?